src/nvim/window.c

```diff
@@ -992,7 +992,7 @@ void ui_ext_win_position(win_T *wp, bool validate)
       String anchor = cstr_as_string((char *)float_anchor_str[c.anchor]);
       ui_call_win_float_pos(wp->w_grid_alloc.handle, wp->handle, anchor,
                             grid->handle, row, col, c.focusable,
-                            wp->w_grid_alloc.zindex);
+                            wp->w_grid_alloc.zindex, c.hide);
```

Comment:
Hmm, what is an external UI expected to do with such a float position when `hide=true` ? I'd just use `ui_call_win_hide()` which UI:s already need to handle when a float is hidden due to being on a different tabpage.